src/solution_part1.kt

```diff
@@ -0,0 +1,88 @@
+import kotlin.math.pow
+
+//## Part I - Practice with Inheritance.
+
+fun main() {
+    //creating a Shape object
+    val s = Shape("Cricle", "white")
+    //printing the object info
+    s.info()
+    println()
+
+    val t = Triangle("Triangle", "Yellow",10.0, 13.0)
+    //printing the object info
+    t.info()
+    //printing the object computeArea
+    println(t.computeArea())
+    println()
+
+    val c = Circle("Circle", "red",20.0)
+    //printing the object info
+    c.info()
+    //printing the object computeArea
+    println(c.computeArea())
+    println()
+
+    val r = Rectangle("Rectangle", "Blue",10.0, 10.0)
```

Comment:
why no comment here

Reply:
line 23: already has a comment which is //printing the object computeArea which will calculate the computeArea of the circle.
line 24: just to print a new line.
line 26: creating a Rectangle object named r that has a name of "Rectangle", it's color is "Blue", and the width value is 10.0 and height 10.0